### PR TITLE
Document --configfile option for dotnet nuget verify

### LINKED ISSUE #2632: Dotnet NuGet verify command need --configFile option
`Dotnet nuget verify` command needs `--configfile` option as feature parity with `nuget verify`.
~~Currently it's being addressed in https://github.com/NuGet/NuGet.Client/pull/4396, once merged we need to document it.~~ 
It's implemented in https://github.com/NuGet/NuGet.Client/pull/4396, now we need to document it.
Ref: https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-nuget-verify

### PR DESCRIPTION
**Pull request by erdembayar**
## Summary

`nuget.exe verify` has configFile option, to reach parity added `configfile` option for `dotnet nuget verify`.

Fixes: https://github.com/NuGet/docs.microsoft.com-nuget/issues/2632

### CODE REVIEW

**Review by JonDouglas — APPROVED** [review]
lgtm

**Review by nkolev92 — APPROVED** [review]

File: docs/core/tools/dotnet-nuget-verify.md [hunk 1]
```diff
@@ -102,3 +105,9 @@ The `dotnet nuget verify` command verifies a signed NuGet package.
   ```dotnetcli
   dotnet nuget verify foo.nupkg --certificate-fingerprint CE40881FF5F0AD3E58965DA20A9F571EF1651A56933748E1BF1C99E537C4E039 --certificate-fingerprint EC10992GG5F0AD3E58965DA20A9F571EF1651A56933748E1BF1C99E537C4E027
   ```
+
+- Verify *foo.nupkg* signature using settings (i.e: `packagesources` , `trustedSigners`) only from specified nuget.config:
```
Comment (tdykstra) [inline docs/core/tools/dotnet-nuget-verify.md:109]:
```suggestion
- Verify the signature of *foo.nupkg* by using settings (`packagesources` and `trustedSigners`) only from the specified *nuget.config* file:
```

**Review by tdykstra — COMMENTED** [review]
Thanks, this looks good, I just had a suggestion to tweak the wording.

File: docs/core/tools/dotnet-nuget-verify.md (hunk 1 above)
Comment (erdembayar) [reply docs/core/tools/dotnet-nuget-verify.md]: Addressed.

**Pull request merged**